values/kube-prometheus-stack.yml

```diff
@@ -45,7 +45,7 @@ grafana:
     storageClassName: ${storage_class_name}
     %{~ endif ~}
     size: ${grafana_pv_size}
-%{~ if prometheus_operator_resources.cpu_requests != "" && prometheus_operator_resources.memory_requests != "" ~}
+%{~ if prometheus_operator_resources.cpu_requests != "" && prometheus_operator_resources.memory_requests != "" }
```

Comment:
```suggestion
%{~ if prometheus_operator_resources.cpu_requests != "" && prometheus_operator_resources.memory_requests != "" ~}
```

Reply:
The ~ has been voluntarily removed, if added in the generation of the values ​​"prometheusOperator" it goes up by one row configuring itself in the size of grafana, thus doing terraform apply fails